src/v0/destinations/snowpipe_streaming/transform.js

```diff
@@ -1 +1,26 @@
-module.exports = require('../snowflake/transform');
+const transform = require('../snowflake/transform');
+const { processWarehouseMessage } = require('../../../warehouse');
+
+const provider = 'snowpipe_streaming';
+
+function process(event) {
+  const whSchemaVersion = event.request.query.whSchemaVersion || 'v1';
+  const whStoreEvent = event.destination.Config.storeFullEvent === true;
+  const destJsonPaths = event.destination?.Config?.jsonPaths || '';
+  return processWarehouseMessage(event.message, {
+    metadata: event.metadata,
+    whSchemaVersion,
+    whStoreEvent,
+    getDataTypeOverride: transform.getDataTypeOverride,
+    provider,
+    sourceCategory: event.metadata ? event.metadata.sourceCategory : null,
```

Comment:
```suggestion
    sourceCategory: event.metadata ? event.metadata?.sourceCategory : null,
```

Reply:
Since we are already checking for `event.metadata` first, why do need to have something like `event.metadata?.sourceCategory`?